substrate/bin/node/inspect/src/command.rs

```diff
@@ -26,14 +26,17 @@ use sc_cli::{CliConfiguration, ImportParams, Result, SharedParams};
 use sc_service::Configuration;
 use sp_runtime::traits::Block;
 
+type HostFunctions =
+	(sp_io::SubstrateHostFunctions, sp_statement_store::runtime_api::HostFunctions);
+
```

Comment:
Not sure if we need to support `benchmarking::HostFunctions` here.
We cannot add dep on `node-cli` as there would be cycle.

Reply:
I don't understand, why can you not add the HF here? Should be a dependency on frame-benchmarking, no?

Reply:
yeah, I could. But it is probably not needed.